src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/indexes/createIndex.svelte

```diff
@@ -171,16 +181,8 @@
                 </div>
             </li>
         {/if}
-        <Button
-            text
-            noMargin
-            on:click={() => {
-                newAttr = true;
-                if (selectedAttribute && selectedOrder) {
-                    attributeList.push({ value: selectedAttribute, order: selectedOrder });
-                    selectedAttribute = selectedOrder = '';
-                }
-            }}>
+
+        <Button text noMargin on:click={addAttribute}>
```

Comment:
This button should be disabled when the new attribute `value` and `order` aren't selected (right now is active but does nothing when clicked)